### PR TITLE
changed the plugin build path to /mswg/release/ufm/plugins*

### PR DESCRIPTION
**Pull request by shahar-spormas**
## What
* change the official path to be  `/mswg/release/ufm/plugins/*` 
* added key to ufm-appliance04;tmp - this is for @egershonNvidia ci

## Why ?
for elad's ci and the create an official path for plugins 

## How ?
changed it 

## Testing ?
locally

### CODE REVIEW

**Review by lennybe — CHANGES_REQUESTED** [review]

File: .ci/matrix_job_release.yaml [hunk 1]
```diff
@@ -24,10 +24,10 @@ steps:
       pwd
       ls
       whoami
-      mkdir -p /auto/mswg/release/ufm/ufm_sdk_3.0/${Plugin_name}
+      mkdir -p /auto/mswg/release/ufm/plugins/${Plugin_name}
       cd plugins/${Plugin_name}/build
-      bash -x ./docker_build.sh ${PLUGIN_VERSION} /auto/mswg/release/ufm/ufm_sdk_3.0/${Plugin_name}
-      ls /auto/mswg/release/ufm/ufm_sdk_3.0/${Plugin_name}
+      bash -x ./docker_build.sh ${PLUGIN_VERSION} /auto/mswg/release/ufm/plugins/${Plugin_name} 
+      ls /auto/mswg/release/ufm/plugins/${Plugin_name}
```
[the remark this answers is not in the document]
Comment (shahar-spormas) [reply .ci/matrix_job_release.yaml:30]: its for the jenkins console

File: .ci/Dockerfile [hunk 2]
```diff
@@ -25,6 +25,7 @@ RUN apt clean && \
     apt-get update && \
     apt-get install -y docker-ce-cli
 
+RUN echo "ufm-appliance04,10.209.36.63 ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABgQDP+RwBY3YDKRmcpZR8pSenGusCSKRZ3SDfq9ert7JWhP1dHpr3/2xzZwCcNBZ1Cl5VyKxzDAsXE62TU7r9DXRFRq0BrSo6qoPWvRVQTxI8RkOSeb8k5uBPckNo/FwTq/4EKR23msxcpNxoyswqAD91VPWScu9TtXlCiG2hqgAye+9j0f0Agp/eOfGPJcfzJqR0Q5ou2gcD5xo8/LevXhd4Mio7QJXo1RVUZUKNmDG0/MS1XFJdE6+NyCSs4Os+wMTB0UtW8TEzgFS2xpfvS0aueOTBvmJ90RQqkbLWhd+EfFdtQ03f8xZcqL1OjxWZ81c0Fd07ieKJpGgDZuOpqsGUgFb5diagpDaF/+xQ4Ciudw/IfxzolgI5ureYlA8IMb4NQe2DdNoXj4wk0cxTnmwl7rFviCDf++avicQmQdOrRwpRK2vYonw+ediDOCX473gupux0c2WoEv6Pv1vK8LYPCxt7t/GUxNtxx/bsOZc7tQf63eUdqvUv6+qcZxPPClM=" >> ~/.ssh/known_hosts
```
[the remark this answers is not in the document]
Comment (shahar-spormas) [reply .ci/Dockerfile:29]: tmp fix, will try to put it in jenkins secrets now

[the remark this answers is not in the document]
Comment (shahar-spormas) [reply .ci/Dockerfile:29]: changed it to mount

File: .ci/matrix_job_release.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (lennybe) [reply .ci/matrix_job_release.yaml:30]: if ./docker_build.sh will fail, you will not know about it.

**Review by lennybe — CHANGES_REQUESTED** [review]

File: .ci/matrix_job_release.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (shahar-spormas) [reply .ci/matrix_job_release.yaml:35]: NP, added an exit code checker

[the remark this answers is not in the document]
Comment (shahar-spormas) [reply .ci/matrix_job_release.yaml:34]: now it exit with the build code as we talked about f2f

File: .ci/matrix_job_release.yaml [hunk 3]
```diff
@@ -20,14 +20,19 @@ steps:
   - name: Build Plugin
     agentSelector: "{nodeLabel: 'SWX-CI-DOCKER'}"
     run: |
+      if [ -e "/auto/mswg/release/ufm/plugins/${Plugin_name}/ufm-plugin-${Plugin_name}_${PLUGIN_VERSION}*" ];then 
+        exit 1
```
Comment (lennybe) [inline .ci/matrix_job_release.yaml:24]: please print the reason for the failure before you exit.
you want to see and not guess the reason of the failure

File: .ci/matrix_job_release.yaml [hunk 4]
```diff
@@ -20,14 +20,20 @@ steps:
   - name: Build Plugin
     agentSelector: "{nodeLabel: 'SWX-CI-DOCKER'}"
     run: |
+      if [ -e "/auto/mswg/release/ufm/plugins/${Plugin_name}/ufm-plugin-${Plugin_name}_${PLUGIN_VERSION}*" ];then 
+        echo -e "A path and plugin with this version already exist."
```
Comment (lennybe) [inline .ci/matrix_job_release.yaml:24]: please print the full path.
otherwize you will spend years by guessing what is existing:)

**Review by lennybe — CHANGES_REQUESTED** [review]

**Review by lennybe — APPROVED** [review]

**Pull request merged**